server/src/main/java/org/elasticsearch/index/mapper/ObjectMapper.java

```diff
@@ -450,11 +451,11 @@ public final Dynamic dynamic() {
      * Returns the parent {@link ObjectMapper} instance of the specified object mapper or <code>null</code> if there
      * isn't any.
      */
-    public ObjectMapper getParentObjectMapper(MapperService mapperService) {
+    public ObjectMapper getParentObjectMapper(Function<String, ObjectMapper> getObjectMapper) {
```

Comment:
so funny how we got to the same change around the same day, same time, for different reasons :)

Reply:
Yeah! With almost the same variable names!